LanguageSpec.md

```diff
@@ -116,13 +116,17 @@ So it would be:
 
 ### Place Character
 
-If we wanted to move the character sprite anywhere on the screen, we could just type
+If we wanted to move the character sprite anywhere on the screen with scaling, we could just type
 
-```PLACE [Character] ([Up, Down, Left, Right])```
+```PLACE [Character] ([x, y, Scale-Height, Scale-Width])```
 
-So it would be:
+So if we were to place Alanis in the top left corner of the screen at 50px height and 20px width, we would write
+
+```PLACE [Alanis] (0,0,50,20)```
+
+If we did not want to worry about scaling we could just write:
```

Comment:
I agree with this as I'm planning to make the underlying draw method work with both supplied scale and default scale values when no scale has been supplied.

Reply:
Awesome, glad we are thinking on the same page.